src/domFunctions.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
+export const setPlaceholderText = () => {
+  const input = document.getElementById('searchBar_text');
+  window.innerWidth < 400
+    ? (input.placeholder = 'City, State, Country')
+    : (input.placeholder = 'City, State, Country, or Zip Code');
+};
+
+export const displayError = (headerMsg, srMsg) => {
+  updateWeatherLocationHeader(headerMsg);
+  updateScreenReaderConfirmation(srMsg);
+};
+
+export const displayApiError = (statusCode) => {
+  const properMsg = toProperCase(statusCode.message);
+  updateWeatherLocationHeader(properMsg);
+  updateScreenReaderConfirmation(`${properMsg}. Please try again.`);
+};
+
+const toProperCase = (text) => {
+  const words = text.split(' ');
+  const properWords = words.map((word) => word.charAt(0).toUpperCase() + word.slice(1));
+  return properWords.join(' ');
+};
+const updateWeatherLocationHeader = (message) => {
+  const h1 = document.getElementById('currentForecast_location');
+  h1.textContent = message;
+};
+
+export const updateScreenReaderConfirmation = (message) => {
+  document.getElementById('confirmation').textContent = message;
+};
+
+export const updateDisplay = (weatherJson, locationObj) => {
+  fadeDisplay();
+  clearDisplay();
+  const weatherClass = getWeatherClass(weatherJson.current.weather[0].icon);
+  setBGImage(weatherClass);
+  const screenReaderWeather = buildScreenReaderWeather(
+    weatherJson,
+    locationObj,
+  );
+  updateScreenReaderConfirmation(screenReaderWeather);
+  updateWeatherLocationHeader(locationObj.getName());
+
+  const ccArray = createCurrentConditionsDivs(
+    weatherJson,
+    locationObj.getUnit(),
+  );
+  displayCurrentConditions(ccArray);
+  //   displaySixDayForecast(weatherJson);
```

Comment:
- [ ] It will be good to delete all the commented code in your project. This will keep your code clean and professional. Remember you can always use git history to go back to the previous version of your code. ⬇️ 
<img width="792" alt="no_inline_comments" src="https://user-images.githubusercontent.com/32361963/126021243-5395e520-a003-4096-bf15-cc7ffe14cd60.png">